docs/rst/manual/theory/ensemble_based_methods.rst

```diff
@@ -397,9 +397,97 @@ Iterative Ensemble Smoother - Ensemble subspace version
+linear in :math:`\psi` is performed by the Kalman methods by employing the following equations
 
+.. math::
+   \begin{align}
+   \mu_{\psi|d} &= \mu_{\psi} + K(d-M\mu_{\psi}),\\
+   \Sigma_{\psi|d} &= (I-KM) \Sigma_{\psi}
+   \end{align}
+
+where
+
+.. math::
+   \begin{align}
+   K = \Sigma_{\psi}M^\top (M\Sigma_{\psi}M^\top + \Sigma_{d})^{-1}
+   \end{align}
+
+is called tha Kalman gain, and :math:`M` is the linear measurement matrix, so that
+
+.. math::
+   \begin{align}
+   \hat{d} = M\mu_{\psi}
+   \end{align}
+
+is the best estimate of :math:`d` under the prior knowledge, and the error is assumed Gaussian with covariance :math:`\Sigma_d`. 
+The ensemble variants draw an :math:`N`-sample :math:`\{\psi\}_{i=1}^N` from the prior, 
+and perturbs observations :math:`d` using the distributions of measurements creating a corresponding observation-sample :math:`\{d\}_{i=1}^N`.
+The perturbations are guaranteed to sum to zero over the sample.
+A posterior sample is then formed from updating the prior with the equation for the posterior mean above
+
+.. math::
+   \begin{align}
+   \{\psi_i | d_i\} = \psi_i + \hat{K}(d_i-M \psi_i),
+   \end{align}
+
+where the estimated Kalman gain :math:`\hat{K}` is found by exchanging the prior covariance with an estimate based on its sample.
+Thus, the ensemble methods combine a sample from the prior with a sample from the likelihood of observed data, to form a new sample from the posterior.
+The posterior distribution that the posterior sample is conseptually sampled from, has mean and covariance found by
+
+.. math::
+   \begin{align}
+   \hat{\mu}_{\psi|d} &= \bar{\psi} + \hat{K}(\bar{d}-M\bar{\psi}),\\
+   \hat{\Sigma}_{\psi|d} &= (I-\hat{K}M) \hat{\Sigma}_{\psi}
+   \end{align}
+
+From this, it is seen that when the sample size tends to infinity and estimates converge to the corresponding population quantities,
+then the ensemble variants converge to the standard Kalman filter in the linear Gaussian case.
+This convergence is however of a stochastic nature.
+
+More deterministic properties of the posterior are observed when the belief in measurements :math:`d` is varied. 
+Intuitively, when measurements have zero belief, i.e. unbounded variance, then the posterior should equal the prior.
+At the other end of the spectrum, if the measurements are perfect with zero variance,
+then the posterior estimate should equal the maximum-likelihood estimate, corresponding to a flat prior, 
+and as we are certain of the belief in this estimate (because the measurements are so amazing), 
+the determinant of the posterior covariance tends to zero from above.
+The maximum likelihood estimate is found by minimizing the relevant part of the negative log-likelihood of the data
+
+.. math::
+   \begin{align}
+   \hat{\mu}_{ml} = \arg\min_{\mu} |d-M\mu|_2
+   \end{align}
+
+Furthermore, for a monotone sequence in belief in measurements, a monotone sequence
```

Comment:
I have a bit of a hard time parsing this sentence.

What if we write the following instead:

Furthermore, for a strictly decreasing sequence in belief in measurements, we expect the distance between the posterior and the maximum likelihood estimate to be strictly decreasing as well.

Reply:
I like the suggestion, except for the words "we expect". It either makes it sound like it is some stochastic convergence with an expectation so and so, or that we are not certain. How about skipping "we expect", and adding "... estimate _will_ be strictly decreasing ..." later on?

Reply:
This is what happens when you say expect to a statistician...

"will be strictly decreasing" is nice,.

Reply:
The other day some friends discussed if they wanted pass/fail or graded exams. They concluded that it was dependent on expected grades and the existing GPA of past exams. For the case of perfect GPA, they said it did not matter as long as the expected grade was perfect. I pointed out that the only way the actual expectation is the perfect grade, is if other grades have zero probability, and thus, you not only _feel and expect_ that you will get a perfect grade but you are certain with zero variation.

Reply:
I make parties fun 🎈